app/templates/components/ui-table.hbs

```diff
@@ -1,6 +1,8 @@
 {{#if isLoading}}
   <div class="ui active inverted dimmer">
-    <div class="ui small text loader">Loading</div>
+    <div class="ui small text loader">
+      Loading
+    </div>
```

Comment:
Just a query, what does this really change?

Reply:
@uds5501 I guess my text editor formatted that automatically. Will correct it.